src/DataFixtures/ORM/LoadManagedUserData.php

```diff
@@ -146,6 +146,7 @@ public function getDependencies()
     {
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`